project/src/test/java/com/google/sps/test/PlacesFetcherTest.java

```diff
@@ -0,0 +1,177 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.test;
+
+import com.google.appengine.repackaged.com.google.common.collect.ImmutableList;
+import com.google.maps.errors.ApiException;
+import com.google.maps.model.Geometry;
+import com.google.maps.model.LatLng;
+import com.google.maps.model.PlaceDetails;
+import com.google.maps.model.PlacesSearchResult;
+import com.google.maps.model.PriceLevel;
+import com.google.sps.data.Place;
+import com.google.sps.data.PlacesFetcher;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.spy;
```

Comment:
Please decide whether you `import static` all mockito options (when, mock, etc.) or none.

Reply:
I don't quite understand, can you please elaborate on that?
I should import options I am not using?